roles/telemetry_verify_metrics/tasks/verify_mariadb_metrics.yml

```
@@ -0,0 +1,54 @@
+- name: Verify mariadb scrapeconfig exists
+  ansible.builtin.include_role:
+    name: common
+  vars:
+    common_cr_test_id: RHOSO-xxxx
+    common_cr_list:
+      - kind: scrapeconfigs.monitoring.rhobs
+        name: telemetry-mysqld-exporter
+
+- name: Get galera CR names
+  ansible.builtin.shell: |
+    oc get galeras -o custom-columns=NAME:.metadata.name --no-headers
+  register: cr_names
+  changed_when: false
+  failed_when: cr_names.rc >= 1 or cr_names.stdout == ""
+
+- name: Create galera pod list
+  ansible.builtin.set_fact:
+    galera_pod_list: "{{ [item + '-galera-0'] }}"
+  loop: "{{ cr_names.stdout_lines }}"
+ 
+- name: List galera pod names
+  ansible.builtin.debug:
+    msg: "galaria pod name ... {{ item }}"
+  loop: "{{ galera_pod_list }}"
+
+- name: Verify galera and mysqld-exporter pods are running
+  ansible.builtin.include_role:
+    name: common
+  vars:
+    common_pod_test_id: RHOSO-xxxx
+    common_pod_status_str: "Running"
+    common_pod_nspace: openstack
+    common_pod_list: galera_pod_list + ['mysqld-exporter-0']
```

I think this needs to go into double curly braces
```suggestion
    common_pod_list: "{{ galera_pod_list + ['mysqld-exporter-0'] }}"
```